services/user.py

```diff
@@ -0,0 +1,56 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
```

Comment:
You import `User` model directly from `db.models`, but then you never use it. Instead, you use `get_user_model()`. It's good that you use `get_user_model()`, since it's a recommended way to operate with User model in Django, but remove the unused import.

Reply:
I'm using User as return annotation in return value